PackageViewModel/PackageViewModel.cs

```diff
@@ -28,6 +28,7 @@ public sealed class PackageViewModel : ViewModelBase, IDisposable
         private readonly IList<Lazy<IPackageRule>> _packageRules;
         private readonly ISettingsManager _settingsManager;
         private readonly IUIServices _uiServices;
+        private readonly CredentialPublishProvider _credentialPublishProvider;
```

Comment:
Should this be an interface?

```c#
public interface ICredentialPublishProvider
{
    string PersonalAccessToken { get; set; }
}
```

Reply:
What other scenarios do you see where the interface makes sense.

Reply:
Currently no one 😄

Reply:
I'd say let's leave it as-is for now and we can always extract an interface if there's a reason to